api/requirements.txt

```
@@ -89,7 +89,7 @@ typing-extensions==4.8.0
     #   pydantic
     #   pydantic-core
     #   sqlalchemy
-urllib3==2.0.7
+urllib3==2.1.0
```

Downgrade urllib3 to meet the requirements of botocore.

```suggestion
urllib3==2.0.7
```